#160703227 update order status

nwashangai opened the pull request:
#### What does this PR do?
it adds update order functionality
#### Description of Task to be completed?
* create endpiont to update status
* write test cases for this endpoint
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
[#160703227](https://www.pivotaltracker.com/story/show/160703227)
#### Screenshots (if appropriate)
* None
#### Questions:
* None

nwashangai committed:
ft: update order status
    - create endpiont to update status
    - write test cases for this endpoint
    [Finishes #160703227]

nwashangai committed:
ft: admin gets specific order
    - create endpoint to get specific order
    - write test cases for this endpoint
    [Finishes #160703172]

dogoyaro commented on src/models/OrderModel.js:
```diff
@@ -17,6 +17,12 @@ class userModel {
         return query(queryString, payload);
     }
+
+    updateOrder(id, status) {
```
Why are these not in an `orderModel` instead?

nwashangai replied:
That's where it is

nwashangai committed:
ft: admin gets all orders
    - create endpoint to get all orders
    - write test cases for this endpoint
    [Finishes #160703159]

nwashangai committed:
ft: adds update menu item functionalities
    - create endpoint to update menu item
    - write test cases for this endpoint
    [Finishes #160703481]

nwashangai committed:
Merge pull request #42 from nwashangai/ft-update-menu-item-2197024

#160703481 adds update menu item functionalities

nwashangai committed:
Merge pull request #41 from nwashangai/ft-get-all-orders-2197024

#160703159 admin gets all orders

nwashangai committed:
Merge pull request #40 from nwashangai/ft-get-specific-2197024

#160703172 admin gets specific order
